module/Activity/src/Activity/Form/ActivityCalendarOption.php

```diff
@@ -72,42 +75,47 @@ public function getInputFilterSpecification()
                         'name' => 'callback',
                         'options' => [
                             'messages' => [
-                                \Zend\Validator\Callback::INVALID_VALUE =>
+                                Callback::INVALID_VALUE =>
                                     $this->translator->translate('The activity must start before it ends'),
                             ],
-                            'callback' => ['Activity\Form\ActivityCalendarOption', 'beforeEndTime']
+                            'callback' => function ($value, $context = []) {
+                                return $this->beforeEndTime($value, $context);
+                            }
                         ],
                     ],
                     [
                         'name' => 'callback',
                         'options' => [
                             'messages' => [
-                                \Zend\Validator\Callback::INVALID_VALUE =>
-                                    $this->translator->translate('The activity must after today'),
+                                Callback::INVALID_VALUE =>
+                                    $this->translator->translate('The activity must start after today'),
                             ],
-                            'callback' => ['Activity\Form\ActivityCalendarOption', 'isFutureTime']
+                            'callback' => function ($value, $context = []) {
+                                return $this->isFutureTime($value, $context);
+                            }
                         ],
                     ],
+//                    [
+//                        'name' => 'callback',
+//                        'options' => [
+//                            'messages' => [
+//                                \Zend\Validator\Callback::INVALID_VALUE =>
+//                                    $this->translator->translate('The activity must be within the given period'),
+//                            ],
+//                            'callback' => function ($value, $context = []) {
+//                                return $this->cannotPlanInPeriod($value, $context);
+//                            }
+//                        ],
+//                    ],
```

Comment:
This should probably be removed, instead of it being a comment?

Reply:
There was some bug in it, but I'm planning on fixing that later

Reply:
So, it is code that is not yet finished, but should be finished before deployment?

Reply:
Not neccesarily

Reply:
I actually think this is working, so I'm uncommenting it